extensions/wikia/PortabilityDashboard/SpecialPortabilityDashboardController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class SpecialPortabilityDashboardController extends WikiaSpecialPageController {
+	const SPECIAL_INSIGHTS_PATH = '/wiki/Special:Insights/';
+	const SPECIAL_INSIGHTS_TYPELESS_TEMPLATE_PAGE = 'templateswithouttype';
+	const SPECIAL_CUSTOM_INFOBOXES_PAGE = 'nonportableinfoboxes';
+
+
+	public function __construct() {
+		parent::__construct( 'PortabilityDashboard', 'pidash', true );
+	}
+
+	public function index() {
+		$model = new PortabilityDashboardModel();
+
+		Wikia::addAssetsToOutput( 'special_portability_dashboard_scss' );
+
+		// template model 
+		$this->response->setVal( 'list', $model->getList() );
+
+		// template helpers
+		$this->response->setVal(
+			'typelessTemplatesInsightsPath', self::SPECIAL_INSIGHTS_PATH . self::SPECIAL_INSIGHTS_TYPELESS_TEMPLATE_PAGE
```

Comment:
I'm not fully on board with this approach; Could we use GlobalTitle to generate full path to the destination page?